pkg/sql/opt/exec/execbuilder/testdata/distsql_merge_join

```diff
@@ -678,14 +678,10 @@ EXPLAIN (DISTSQL)
 distribution: full
 vectorized: true
 ·
-• merge join
-│ equality: (pid1, cid2, cid3) = (pid1, cid2, cid3)
-│ left cols are key
-│
-├── • scan
-│     missing stats
-│     table: child2@primary
-│     spans: FULL SCAN
+• lookup join
```

Comment:
This plan and some others changed from merge joins to lookup joins. It appears that the tests were specially for merge joins, so maybe I need to change the queries? However, this file looks all related to interleaved tables, which are deprecated, so perhaps its not worth the effort?